addons/web/static/src/views/pivot/pivot_view.scss

```diff
@@ -21,79 +21,42 @@
             padding: 0!important;
```

Comment:
Not a big fan of all these CSS resets :/
`bg-white border-0 p-0` classes?

Reply:
After checking, this class was not used anymore so I removed it 👍